cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
Can you offer a design sketch on how the archive and exec APIs can be used to do two-way sync for real world apps and development workflows?